Dockerfile

```
@@ -1,10 +1,10 @@
 FROM rust:latest as builder
 
 RUN apt-get update && apt-get install -y libfontconfig1-dev libgraphite2-dev libharfbuzz-dev libicu-dev zlib1g-dev
-RUN cargo install tectonic --force --vers 0.1.11
+RUN cargo install tectonic --force --vers 0.3.0
```

```suggestion
RUN cargo install tectonic --force --vers 0.3.1
```
tectonic 0.3.1 is out since a few days :)